src/containers/NewItem/index.js

```diff
@@ -1,12 +1,132 @@
 import React, { Component } from 'react';
+import {connect} from 'react-redux';
+import { withRouter} from 'react-router-dom';
+import {createItem} from '../../actions/itemActions';
 
 class NewItem extends Component {
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
```

Comment:
May need to do the same here:
Change the item keys (and check in ItemList) to: 
{
id,
name,
price,
model,
dimensions,
created_at,
updated_at,
notes,
user_id,
category_id,
condition_id,
item_status_id,
image
}